### DISCUSSION TITLE
DoPolarizationSums

### DISCUSSION THREAD

**Discussion by sunnyasquabla** [FeynCalc/feyncalc#267] [Q&A]
Hi,
I am trying to evaluate the decay rate for H to Z \gamma.
I calculated the amplitude, now I want to do the polarization sums. I want to know whether this is correct.

![image](https://github.com/FeynCalc/feyncalc/assets/107980747/d35c5e6d-28a4-4674-bbfd-05b132b7bd0b)

k1 is the momentum of photon,
k2 is of Z-boson
scalarLoop5 is the amplitude for this process.

Thanks.

**Comment by vsht** [comment]
Hi,

for the photon polarization sum I'd use the formula with an auxiliary
vector, you can find that in the manual of DoPolarizationSums and 
PolarizationSum.

Apart from that, to check the correctness you need to compare your 
result with the literature. There's no other way.

Cheers,
Vladyslav

Am 02.05.24 um 12:52 schrieb Sunny:
